diff --git a/node_modules/openapi-trpc/dist/openapi-trpc.d.ts b/node_modules/openapi-trpc/dist/openapi-trpc.d.ts
index 07ca9d7..ab7ee70 100644
--- a/node_modules/openapi-trpc/dist/openapi-trpc.d.ts
+++ b/node_modules/openapi-trpc/dist/openapi-trpc.d.ts
@@ -1,14 +1,12 @@
 import { OpenAPIV3 } from 'openapi-types';
-import { RootConfig } from '@trpc/server';
-import { Router } from '@trpc/server';
-import { RouterDef } from '@trpc/server';
+import { RouterDef, AnyRouter, Router } from '@trpc/server/unstable-core-do-not-import'
 
 declare const allowedOperationKeys: ("tags" | "summary" | "description" | "externalDocs" | "deprecated")[];
 
 /**
  * @public
  */
-export declare function generateOpenAPIDocumentFromTRPCRouter<R extends Router<any>>(inRouter: R, options?: GenerateOpenAPIDocumentOptions<MetaOf<R>>): OpenAPIV3.Document<{}>;
+export declare function generateOpenAPIDocumentFromTRPCRouter<R extends Router<any, any>>(inRouter: R, options?: GenerateOpenAPIDocumentOptions<MetaOf<R>>): OpenAPIV3.Document<{}>;
 
 /**
  * @public
@@ -18,7 +16,7 @@ export declare interface GenerateOpenAPIDocumentOptions<M extends OperationMeta>
     processOperation?: (operation: OpenAPIV3.OperationObject, meta: M | undefined) => OpenAPIV3.OperationObject | void;
 }
 
-declare type MetaOf<R extends Router<any>> = R extends Router<RouterDef<infer D, any>> ? D extends RootConfig<infer C> ? C['meta'] : never : never;
+declare type MetaOf<TRouter extends AnyRouter> = TRouter extends { _def: { meta: infer M } } ? M : never;
 
 /**
  * @public
